frontend/simple/controller/router.js

```diff
@@ -209,6 +211,39 @@ var router = new Router({
       },
       beforeEnter: createEnterGuards(loginGuard)
     },
+    {
+      path: '/messages',
+      name: 'Messages',
+      component: Messages,
+      meta: {
+        title: 'Messages'
+      },
+      beforeEnter: createEnterGuards(loginGuard) // NOTE comment this to test on a iPhone
+    },
+    {
+      path: '/messages/:name',
+      name: 'MessagesConversation',
+      component: Messages,
+      beforeEnter: createEnterGuards(loginGuard) // NOTE comment this to test on a iPhone
+      // BUG/REVIEW "CANNOT GET /:username" when username has "." in it
+      // ex: messages/joe.kim doesnt work but messages/joekim works fine.
+      // Possible Solution: https://router.vuejs.org/guide/essentials/history-mode.html#example-server-configurations
```

Comment:
See my comments down below about how `$route.params.` is being used incorrectly in general. So first fix that, and if there's still a problem, you can use query parameters to pass in the username.